### PR TITLE
te-36 update composer.json

### PR DESCRIPTION
**Pull request by dereuromark**
Do we still want this?

### CODE REVIEW

**Comment by dereuromark** [comment]
where did the autoload dev go?

**Comment by zyuzka** [comment]
> where did the autoload dev go?

Lol. Merge was without any conflicts

File: composer.json [hunk 1]
```diff
@@ -29,15 +19,27 @@
...
+  "keywords": [
+    "framework",
+    "sniffer",
+    "architecture",
+    "standards"
+  ],
+  "authors": [
+    {
+      "name": "Spryker",
+      "homepage": "http://spryker.com"
```
Comment (dereuromark) [inline composer.json]: should  be https @stereomon

File: composer.json [hunk 2]
```diff
@@ -1,19 +1,9 @@
 {
   "name": "spryker/architecture-sniffer",
-  "description": "Spryker Code Architecture Sniffer",
   "type": "library",
-  "license": "MIT",
-  "keywords": ["framework", "sniffer", "architecture", "standards"],
+  "description": "architecture-sniffer module",
```
Comment (dereuromark) [inline composer.json]: name must be reverted. it is not a module

**Pull request merged**